Assets/HotmaxEditorVR/propertiesMenu/propertiesMenu.cs

```diff
@@ -0,0 +1,207 @@
+            foreach (Renderer renderer in renderers)
+            {
+                materialNames.Add(renderer.material.name);
+            }
+
+            removeLastMaterialPage();
+
+            //get only unique instances of the material names
+            materialNames = materialNames.Distinct().ToList();
+
+            //create "chunks" groups of 4 to display the object
+            chunkedMaterialList = ListExtensions.ChunkBy(materialNames, 4);
+            showMaterialPage(0);
+
+            //--------------------------------------MASS--------------------------------------\\
+
+            //--GETTING
+            //if it has a Rb it should be under 90 mass units
+            if (_selectedObject.GetComponent<Rigidbody>())
+            {
+                selectedObjectRb = _selectedObject.GetComponent<Rigidbody>();
+
+                if (selectedObjectRb.mass < 60)
+                {
+                    physicsText.text = "#grabbable #throwable";
+                }
+                else if (selectedObjectRb.mass > 60 && selectedObjectRb.mass < 90)
+                {
+                    physicsText.text = "#heavy #non-grabbable";
+                }
+
+                slider.transform.localPosition = new Vector3(0, 0, selectedObjectRb.mass / 10);
+            }
+            else
+            {
+                slider.transform.localPosition = new Vector3(0, 0, 10);
+                physicsText.text = "#non-moveable";
+            }
+        }
+    }
+
+    public void materialPageUp()
+    {
+        if (currentMaterialsPage >= chunkedMaterialList.Count - 1)
+        {
+            return;
+        }
+        else
+        {
+            currentMaterialsPage += 1;
+            removeLastMaterialPage();
+            showMaterialPage(currentMaterialsPage);
+        }
+    }
+
+    public void materialPageDown()
+    {
+        if (currentMaterialsPage <= 0)
+        {
+            return;
+        }
+        else
+        {
+            currentMaterialsPage -= 1;
+            removeLastMaterialPage();
+            showMaterialPage(currentMaterialsPage);
+        }
+    }
+
+    public void updatePhysicsInfo()
+    {
+        //TODO: move this to a slider class mutate, these properties and animations will probably have to go through the state
+
+        //--TODO: needs to be a public method for the trigger unclick of the slider
+        if(_selectedObject != null)
+        {
+            sliderUnit = slider.transform.localPosition.z;
+
+            if (sliderUnit < 6)
+            {
+                if (!_selectedObject.GetComponent<Rigidbody>())
+                {
+                    _selectedObject.AddComponent<Rigidbody>();
+                }
+                _selectedObject.GetComponent<Rigidbody>().mass = sliderUnit * 10;
+                _selectedObject.GetComponent<Rigidbody>().isKinematic = true;
+                physicsText.text = "#grabbable #throwable";
+
+            }
+            else if (sliderUnit > 6 && sliderUnit < 9)
```

Comment:
what if sliderUnit is 6?

Reply:
got it, changed it to >= 6 on the first statement